templates/pipeline.jinja.html

```diff
@@ -773,12 +773,21 @@ <h1>
 
 
           {% if job["stdout"] %}
-          <div class="output-box"><xmp>stdout:
-            {% for line in job["stdout"] %}
-{{ line.strip() }}
-            {%- endfor %}{# line in job["stdout"] #}
-            </xmp>
-          </div><!-- class="output-box" -->
+            {% if job["wrapper_arguments"]["tags"] is not none and "literal-stdout" in job["wrapper_arguments"]["tags"] %}
+            <div class="pseudo">
```

Comment:
what is `pseudo`? You haven't added any particular style for it?

Reply:
`pseudo` sounds like it would be a wrapper to group whatever elements the user wants to render. Maybe the div should be given an `id=literal-stdout` tag?

Reply:
There is one of this div per job on the pipeline page, so we can't give it an `id` but I agree with `class=literal-stdout`.

But I understand, now, of course there's no style because there isn't really any style that we could reasonably give to it. The stdout is expected to provide its own styling.

Perhaps, if anything, the div could have a special border to indicate that everything inside it is program output rather than something that litani generated directly? Like a border with a header at the top that says "literal stdout"? Though without a border it probably looks cleaner.